### ISSUE TITLE
Incompatible with flutter 2.8?

### ISSUE THREAD

**Issue by abel-123** [inapay/iban_form_field#7]
After upgrading to flutter 2.8, the following error is thrown when attempting to run the application:

```
../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/iban_form_field-0.2.1/lib/src/iban_form_field.dart:194:19: Error: The method 'WhitelistingTextInputFormatter' isn't defined for the class '_IbanFormFieldState'.
 - '_IbanFormFieldState' is from 'package:iban_form_field/src/iban_form_field.dart' ('../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/iban_form_field-0.2.1/lib/src/iban_form_field.dart').
Try correcting the name to the name of an existing method, or defining a method named 'WhitelistingTextInputFormatter'.
                  WhitelistingTextInputFormatter(RegExp('[A-Z]')),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/iban_form_field-0.2.1/lib/src/iban_form_field.dart:221:19: Error: The method 'WhitelistingTextInputFormatter' isn't defined for the class '_IbanFormFieldState'.
 - '_IbanFormFieldState' is from 'package:iban_form_field/src/iban_form_field.dart' ('../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/iban_form_field-0.2.1/lib/src/iban_form_field.dart').
Try correcting the name to the name of an existing method, or defining a method named 'WhitelistingTextInputFormatter'.
                  WhitelistingTextInputFormatter(RegExp('[0-9]')),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

The reason seems to be that WhitelistingTextInputFormatter has been deprecated in Flutter 2.8. 
(I believe it can be replace by FilteringTextInputFormatter, but have not looked into this in detail)

**Comment by joeljpresent** [comment]
abel-123 is right: `WhitelistingTextInputFormatter` and other text input formatting classes have been deprecated for some time and [have been completely removed in Flutter 2.8](https://github.com/flutter/flutter/pull/90296).
This means that the package cannot be built on Flutter 2.8.

```dart
@Deprecated(
  'Use FilteringTextInputFormatter.allow instead. '
  'This feature was deprecated after v1.20.0-1.0.pre.',
)
class WhitelistingTextInputFormatter extends FilteringTextInputFormatter {
```

The use of `WhitelistingTextInputFormatter` must be replaced with `FilteringTextInputFormatter.allow`.

**Comment by haarts** [comment]
I've just released version 0.2.2 which fixes this. @sugihiru thank you for your contribution!
